browser/ui/views/split_view/split_view.h

```diff
@@ -84,6 +92,12 @@ class SplitView : public views::View, public SplitViewBrowserDataObserver {
     return secondary_devtools_web_view_;
   }
 
+#if BUILDFLAG(ENABLE_SPEEDREADER)
+  void OnReaderModeToolbarActivate(ReaderModeToolbarView* toolbar) override;
+
+  void UpdateSecondaryReaderModeToolbar(ReaderModeToolbarView* primary_toolbar);
```

Comment:
Instead of getting primary toolbar as an args, I think we could get it from `brave_browser_view_->reader_mode_toolbar()`?